specification/connectedvmware/resource-manager/readme.md

```diff
@@ -56,7 +56,6 @@ swagger-to-sdk:
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-net
   - repo: azure-sdk-for-ruby
-  - repo: azure-sdk-for-net
     after_scripts:
       - bundle install && rake arm:regen_all_profiles['azure_mgmt_connectedvmware']
```

Comment:
This one is not supported by track 2 yet, just remove the duplicate setting which steals the after script that should belong to ruby.

Reply:
We do have a track 2 sdk\connectedvmwarevsphere\Azure.ResourceManager.ConnectedVMwarevSphere, though. Is this the same thing?

Reply:
You're right. I somehow missed that due to the different service directory naming.